api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py

```diff
@@ -369,3 +373,124 @@ def create_multiauthors_books(venues: list[offerers_models.Venue]) -> None:
     author = "collectif"
     for _ in range(3):
         create_offer_with_ean(Fake.ean13(), random.choice(venues), author=author)
+
+
+def create_offers_with_more_extra_data() -> None:
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏le naing est flou, dans 3 mois on aura oublié ce qu'il y a dedans. 
Proposition: (on peut réfléchir à de meilleurs naming pour les sous-fonctions)
```python
def create_offers_with_more_extra_data():
    venue = VenueFactory()
    create_music_offers_with_artist(venue)
    create_book_offers_with_author(venue) # ça existe déjà en masse ça, avec les GTLs, on en a peut être pas besoin
    create_cine_offer_with_cast(venue)
```